docs/components/oculus-touch-controls.md

```diff
@@ -34,34 +34,43 @@ mappings, events, and a Touch controller model.
 | triggerup            | Trigger released.              |
 | triggertouchstart    | Trigger touched.               |
 | triggertouchend      | Trigger no longer touched.     |
+| triggerchanged       | Trigger changed.               |
 | thumbstickdown       | Thumbstick pressed.            |
 | thumbstickup         | Thumbstick released.           |
 | thumbsticktouchstart | Thumbstick touched.            |
 | thumbsticktouchend   | Thumbstick no longer touched.  |
+| thumbstickchanged    | Thumbstick changed.            |
 | gripdown             | Grip button pressed.           |
 | gripup               | Grip button released.          |
 | griptouchstart       | Grip button touched.           |
 | griptouchend         | Grip button no longer touched. |
+| gripchanged          | Grip button changed.           |
 | Adown                | A button pressed.              |
 | Aup                  | A button released.             |
 | Atouchstart          | A button touched.              |
 | Atouchend            | A button no longer touched.    |
+| Achanged             | A button changed.              |
```

Comment:
the ones that correspond to buttons A, B, X, and Y are uppercase for clarity because that is the way the button labels are always used, IMO don't think we should lowercase them?  worried about namespace collisions too.  Note the existing support is the same way...